src/main/java/com/mixpanel/android/mpmetrics/MixpanelActivityLifecycleCallbacks.java

```diff
@@ -36,7 +35,9 @@ public void onActivityDestroyed(Activity activity) { }
     public void onActivitySaveInstanceState(Activity activity, Bundle outState) { }
 
     @Override
-    public void onActivityResumed(Activity activity) { }
+    public void onActivityResumed(Activity activity) {
```

Comment:
That is true. This callback will never happen if AutoShowMixpanelUpdates is set to false, and the user will need to call `joinExperimentIfAvailable` themselves. The intent is to allow folks to install their own loading screens or otherwise customize the UX if they'd like.